bedrock/base/templates/includes/protocol/footer/mozilla.html

```diff
@@ -3,7 +3,7 @@
  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
 
 {% if LANG.startswith('en-') %}
```

Comment:
Some locales have already completed translations, we may be able to add a few to this conditional before it merges. I'll make a list.